### PR TITLE
improved log message on graceful shutdown

### PR DESCRIPTION
**Pull request by celaus**

### CODE REVIEW

File: src/main/java/org/elasticsearch/cluster/routing/allocation/deallocator/AllShardsDeallocator.java [hunk 1]
```diff
@@ -198,9 +198,13 @@ public boolean canDeallocate() {
         for (ObjectObjectCursor<String, IndexMetaData> entry : clusterState.metaData().indices()) {
             if (!localNode.shardsWithState(entry.key, ShardRoutingState.STARTED, ShardRoutingState.INITIALIZING, ShardRoutingState.RELOCATING).isEmpty()) {
                 maxReplicas = Math.max(maxReplicas, entry.value.numberOfReplicas());
+                if(numNodes <= entry.value.numberOfReplicas() + 1) {
```
Comment (chaudum) [inline src/main/java/org/elasticsearch/cluster/routing/allocation/deallocator/AllShardsDeallocator.java]: code style: whitespace

**Comment by chaudum** [comment]
what about the other deallocators?

**Comment by chaudum** [comment]
changes are missing

**Comment by celaus** [comment]
no message required:

```   
   /**
     * This deallocator can always deallocate
     */
    @Override
    public boolean canDeallocate() {
        return true;
    }
```

**Comment by chaudum** [comment]
ack

**Pull request merged**